content/api/commands/window.md

```diff
@@ -115,14 +126,26 @@ if (window.Cypress) {
 Cypress Test Runner can wait for the property `window.appReady` to be `true`
 before every test
 
-```javascript
+:::visit-mount-test-example
```

Comment:
This example should probably just be an E2E test, since it's dealing with a whole app and not a component.